src/actions/clapsAction.js

```diff
@@ -0,0 +1,56 @@
+import typeGenerator from './typeGenerator';
+import request from '../utils/request';
+
+export const clapArticleType = typeGenerator('CLAP_ARTICLE');
+
+/**
+ * Triggers a clap success
+ *
+ * @param {boolean} payload - The user payload state
+ * @returns {object} - Returns an action object
+ */
+export const clapArticleSuccess = payload => ({
+  type: clapArticleType.success,
+  payload,
+});
+
+/**
+ * Triggers a clap failure
```

Comment:
#### Comment
- Do action generators really trigger?

Reply:
Generators dispatches an action type. I'll make an update on that, thanks.